winrm4j/src/test/java/io/cloudsoft/winrm4j/winrm/WinRmToolExecLiveTest.java

```diff
@@ -75,7 +75,39 @@ public void testExecFailingScript() throws Exception {
     @Test(groups="Live")
     public void testExecScriptExit0() throws Exception {
         assertExecSucceeds("exit /B 0", "", "");
-        assertExecSucceeds(ImmutableList.of("exit /B 0"), "", "");
+    }
+
+    @Test(groups = "Live")
+    public void testChainCommands() {
+        assertExecCommand("echo Hi & echo World", "Hi \r\nWorld", "", 0);
+    }
+
+    /**
+     * Please bear in mind this behavior when executing commands.
+     */
+    @Test(groups="Live")
+    public void testExecRNSplitExit() throws Exception {
+        assertExecCommand("echo Hi\r\necho World\r\n", "Hi", "", 0);
```

Comment:
Are you saying that the `echo World` has not been executed? I presume that is the case. Would we be better off having a failing test, and marking that as `enabled=false` (with a comment about why it fails)? Clearly this test is not demonstrating desired behaviour.